doc/concepts/recipe.rst

```diff
@@ -17,17 +17,23 @@ building phase to provide information about cells in the model, such as:
   * **Gap junction connections** on a cell.
   * **Probes** on a cell.
 
+Recipes are structured to provide a consistent interface for describing each cell in the
+network using their global identifier (`gid`).
+To be able to quickly look-up properties related to the connections going in and out of a
```

Comment:
I don't want to nitpick, but I find this sentence quite difficult to read. How about: 

`This allows the simulator to be able to quickly look-up properties related to the connections going in and out of a cell (think of synapses, gap junctions, but also probes and spike inputs); which helps make Arbor fast and easily distributable over many nodes.`